pkgs/build-support/testers/default.nix

```diff
@@ -9,8 +9,14 @@
       version ? package.version,
     }: runCommand "${package.name}-test-version" { nativeBuildInputs = [ package ]; meta.timeout = 60; } ''
       if output=$(${command} 2>&1); then
-        grep -Fw "${version}" - <<< "$output"
-        touch $out
+        if grep -Fw "${version}" - <<< "$output"; then
+          touch $out
+        else
+          echo "Version string '${version}' not found!"
+          echo "The output was:"
+          echo "$output"
+          exit 1
+        fi
       else
         echo "$output" >&2 && exit 1
```

Comment:
Would be also nice to add error message here, the command can fail without output too.

Reply:
like so?